storage/innobase/buf/buf0buf.cc

```diff
@@ -752,14 +753,16 @@ based off https://www.kernel.org/doc/html/latest/accounting/psi.html#pressure-in
 and https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#memory */
 class mem_pressure
 {
-  /* triggers + pipe */
+  /* triggers + eventfd */
   struct pollfd m_fds[3];
   nfds_t m_num_fds;
-  int m_pipe[2]= {-1, -1};
+  int m_event_fd= -1;
 
   std::thread m_thd;
-  /* mem pressure garbarge collection resticted to interval */
+  /* mem pressure garbage collection restricted to interval */
   static constexpr ulonglong max_interval_us= 60*1000000;
+  static bool m_abort;
```

Comment:
If this were declared non-`static` we would not need a separate initialization. Maybe, to silence any ThreadSanitizer warning, this should be `Atomic_relaxed<bool>`.

Reply:
missed the obvious non-static implementation, thanks.